Documentation/AdditionalFunctionality/Jobs.md

```diff
@@ -0,0 +1,132 @@
+
+    func handle() -> String {
+        "Current date: \($dateService.date)"
+    }
+}
+
+var configuration: Configuration {
+    Schedule(DateService.self, on: "10 * * * *")
+}
+```
+
+## Subscribing to events
+
+Besides emitting events, `Job`s can also listen to changes of other [ObservedObjects](<./../Communicational\ Patterns/2.\ Tooling/2.4.\ ObservedObject.md>) which are declared with the `@ObservedObject` property wrapper.
+This will also trigger the `run` method of a `Job`.
+To differentiate between the scheduled execution and the execution of an `ObservableObject`, the projected Boolean value `changed` of the property wrapper can be used. Projected values of property wrappers are accessed using the `$` prefix operator.
+
+Consider the following example which consists of a `Component` called _RegisterComponent_, a `Job` _SummaryJob_, and an `ObservableObject` _VisitorObservedObject_.  
+The _RegisterComponent_ is responsible to count every access to its path and store this information in _VisitorObservedObject_. Using `Environment(keypath, value)` in the `configuration` stored property, which takes as arguments a keypath and the corresponding value, we can declare the _VisitorObservedObject_ as a Singleton. This allows us to use the `ObservableObject` in the web service with the property wrappers `@Environment` and `@ObservedObject` and the keypath. The `Job` will send a weekly push notification based on the total number of visitors and also when someone accessed the _RegisterComponent_ by listening to changes to the _VisitorObservedObject_. To differentiate between these two cases we use the projected value `$visitors.changed` to check if the `Job` was triggered by being scheduled or by the _VisitorObservedObject_.
+
+```swift
+struct TestWebService: WebService {
+    struct VisitorObservedObject: ObservableObject {
+        @Published var count = 0
+    }
+
+    struct SummaryJob: Job {
+        @ObservedObject(\.visitorObservedObject) var visitors: VisitorObservedObject
+
+        @Environment(\.notificationCenter) var notificationCenter: NotificationCenter
+
+        func run() {
+            if $visitors.changed {
+                notificationCenter.send(alert: Alert(title: "We have a new customer"), to: "visitorTopic")
+            } else {
+                notificationCenter.send(alert: Alert(title: "This week we had a total of \(visitorObject.count) visitors", to: "visitorTopic"))
+            }
+        }
+    }
+
+    struct RegisterComponent: Component {
+        @Environment(\.visitorObservedObject) var visitors: VisitorObservedObject
+
+        func handle() -> String {
+            visitors.count += 1
+            return "Welcome to our site!"
+        }
+    }
+
+    var content: some Component {
+        RegisterComponent()
+    }
+
+    var configuration: Configuration {
+        Environment(\.visitorObservedObject, VisitorObservedObject())
+        Schedule(SummaryJob.self, on: "0 9 * * 5")
```

Comment:
I would suggest to let the developer pass in a concrete instance of a `Job` instead of the type. E.g. the `Job` might have an initializer that we are not aware of and we therefore will not be able to create an instance. Or the `Job` has an initializer that takes a parameter to configure the `Job` and we want to schedule the job twice with different parameters.